### PR TITLE
Updates to docker, and warning messages

### PR DESCRIPTION
**Pull request by gagelarsen**
- Add tethys manage.py to docker env
- Updated error/warning message when apps installed before database

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,7 +17,8 @@ ENV  TETHYS_HOME="/usr/lib/tethys" \
      TETHYS_DB_SUPERUSER_PASS="pass" \
      PORTAL_SUPERUSER_NAME="" \
      PORTAL_SUPERUSER_EMAIL="" \
-     PORTAL_SUPERUSER_PASSWORD=""
+     PORTAL_SUPERUSER_PASSWORD="" \
+     TETHYS_MANAGE="/usr/lib/tethys/tethys_portal/manage.py"
```
Comment (swainn) [inline Dockerfile]: Should be ${TETHYS_HOME}/tethys/tethys_portal/manage.py

File: Dockerfile [hunk 2]
```diff
@@ -120,16 +121,18 @@ ADD --chown=www:www tethys_portal ${TETHYS_HOME}/tethys/tethys_portal/
...
 ADD --chown=www:www .git ${TETHYS_HOME}/tethys/.git/
 
 # Run Installer
 RUN /bin/bash -c '. ${CONDA_HOME}/bin/activate ${CONDA_ENV_NAME} \
-  ; python setup.py develop'
-RUN mkdir -p ${TETHYS_PERSIST}
-RUN mkdir ${APPS_ROOT} ${WORKSPACE_ROOT} ${STATIC_ROOT}
+  ; python setup.py install'
+RUN /bin/bash -c '. ${CONDA_HOME}/bin/activate ${CONDA_ENV_NAME} \
+  ; tethys gen settings'
```
Comment (swainn) [inline Dockerfile]: Add a comment explaining that we need to have a settings.py file for some commands to work in the Dockerfile (like tethys install)

File: Dockerfile [hunk 3]
```diff
@@ -120,16 +121,18 @@ ADD --chown=www:www tethys_portal ${TETHYS_HOME}/tethys/tethys_portal/
...
+RUN mkdir -p ${TETHYS_PERSIST}; mkdir ${APPS_ROOT} ${WORKSPACE_ROOT} ${STATIC_ROOT}
```
Comment (swainn) [inline Dockerfile]: Combine into single `mkdir` command. Be sure to have `-p` option

**Comment by sdc50** [comment]
> Get coverage to 100%

@swainn Looks like this PR is ready to go. I've approved, so once you update your review status I think we can merge.

**Pull request merged**